main.tf

```diff
@@ -47,40 +71,14 @@ data "aws_region" "current" {}
 # The AWS account id
 data "aws_caller_identity" "current" {}
 
-# JSON template defining all the access controls to allow AWS services to write
-# to this bucket
-data "template_file" "aws_logs_policy" {
-  template = "${file("${path.module}/policy.tpl")}"
-
-  vars = {
-    region                  = "${var.region}"
-    bucket                  = "${var.s3_bucket_name}"
-    default_allow           = "${var.default_allow}"
-    allow_cloudtrail        = "${var.allow_cloudtrail}"
-    cloudtrail_logs_prefix  = "${var.cloudtrail_logs_prefix}"
-    allow_cloudwatch        = "${var.allow_cloudwatch}"
-    cloudwatch_logs_prefix  = "${var.cloudwatch_logs_prefix}"
-    allow_config            = "${var.allow_config}"
-    config_logs_prefix      = "${var.config_logs_prefix}"
-    allow_elb               = "${var.allow_elb}"
-    elb_log_account_arn     = "${data.aws_elb_service_account.main.arn}"
-    elb_logs_prefix         = "${var.elb_logs_prefix}"
-    allow_alb               = "${var.allow_alb}"
-    alb_logs_prefix         = "${var.alb_logs_prefix}"
-    allow_redshift          = "${var.allow_redshift}"
-    redshift_log_account_id = "${data.aws_redshift_service_account.main.id}"
-    redshift_logs_prefix    = "${var.redshift_logs_prefix}"
-  }
-}
-
 #
 # S3 Bucket
 #
 
 resource "aws_s3_bucket" "aws_logs" {
   bucket = "${var.s3_bucket_name}"
 
-  acl    = "${var.default_allow || var.allow_s3  ? "log-delivery-write" : "private"}"
+  acl    = "${var.default_allow || var.allow_s3 ? "log-delivery-write" : "private"}"
```

Comment:
Not clear what is happening here. Add a comment "Allows other s3 buckets to write to this s3 bucket".